### PR TITLE
[ICP-7227] Fibaro Smoke Sensor is pairing as flood sensor.

### PR DESCRIPTION
**Pull request by luisferreira1**
Change-Id: I5fd65dc644f60489a00191c91662fc5d02e9df0a

### CODE REVIEW

**Comment by luisferreira1** [comment]
Raw Description:
zw:Ss type:0701 mfr:010F prod:0C02 model:3002 ver:3.03 zwv:4.05 lib:03 cc:5E,86,72,59,73,80,22,31,56,98,7A sec:20,5A,85,84,71,70,8E,9C role:06 ff:8C01 ui:8C01

**Comment by tpmanley** [comment]
Can you point this PR against staging? We'll want to get it in next week's release if not sooner

**Comment by luisferreira1** [comment]
> Can you point this PR against staging? We'll want to get it in next week's release if not sooner

I can, but I don't know how. I never had to do that before.

**Comment by tpmanley** [comment]
@luisferreira1 , I'll reach out in Slack

File: devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy [hunk 1]
```diff
@@ -23,6 +23,7 @@ metadata {
 		capability "Temperature Alarm"
 
 		fingerprint mfr:"010F", prod:"0C02", model:"1002"
+		fingerprint mfr:"010F", prod:"0C02", model:"3002", deviceId: "0x0701", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x80,0x22,0x31,0x56,0x98,0x7A"
```
Comment (greens) [inline devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy]: just include the mfr, prod, and model fields.

Those should be enough to uniquely specify the device

Comment (luisferreira1) [reply devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy]: Ok, changes done. Please check @greens .

**Review by greens — APPROVED** [review]

**Pull request merged**